k8s/tyk-stack-ingress/k6.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: k6-test-script
+  namespace: tools
+data:
+  test-script.js: |
+    import http from 'k6/http';
+    import { sleep, check } from 'k6';
+    
+    // Get environment variables with defaults
+    const API_NAME = __ENV.API_NAME || 'test';
+    const TEST_DURATION = __ENV.TEST_DURATION || '30s';
+    const TARGET_NAMESPACE = __ENV.TARGET_NAMESPACE || 'tyk-dp-1';
+    
+    export const options = {
+      vus: 10,
+      duration: TEST_DURATION,
+      thresholds: {
+        http_req_failed: ['rate<0.01'],
+        http_req_duration: ['p(95)<500']
+      },
+      // Add summary export for better output
+      summaryTrendStats: ['avg', 'min', 'med', 'max', 'p(90)', 'p(95)', 'p(99)']
+    };
+
+    export default function() {
+      const res = http.get(`http://gateway-svc-tyk-data-plane-tyk-gateway.${TARGET_NAMESPACE}.svc.cluster.local:8080/${API_NAME}/upstream`);
+      check(res, {
+        'status is 200': (r) => r.status === 200,
+      });
+      sleep(1);
+    }
```

Comment:
it might be better to keep these scripts as separate files rather than inlining it in configmap definition considering maintenance

Reply:
[mccutchen/go-httpbin](https://github.com/mccutchen/go-httpbin) has 40MB and my flask app was 150MB.
I've changed it to use go-httpbin. Thanks for the comment